src/services/conseillers/conseillers.repository.ts

```diff
@@ -204,9 +210,19 @@ const filterIsRuptureMisesEnRelationStructure = (
       };
     default:
       return {
-        $and: [
-          { statut: { $in: ['nouvelle_rupture', 'finalisee'] } },
-          { 'conseiller.$id': { $in: conseillerIdsRecruter } },
+        $or: [
+          {
+            $and: [
+              { statut: { $eq: 'finalisee_rupture' } },
+              { 'conseiller.$id': { $in: conseillerIdsRupture } },
+            ],
+          },
+          {
+            $and: [
+              { statut: { $in: ['nouvelle_rupture', 'finalisee'] } },
```

Comment:
on peut pas regrouper en un  ?  { statut: { $in: ['nouvelle_rupture', 'finalisee', 'finalisee_rupture'] }